### DISCUSSION TITLE
Lettuce Migration

### DISCUSSION THREAD

**Discussion by raghuchowdary1994** [redis/lettuce#2204] [Q&A]
Hello Team,

We are currently on the Lettuce 4.3.0
I am looking for release notes to migrate Lettuce from 4.x to 5.x and from 5.x to 6.x 

Can you guys share if there are any migration information ?

**Comment by mp911de** [comment]
Check out the what's new section to learn about the changes: https://lettuce.io/core/5.3.7.RELEASE/reference/index.html#new-features.5-0-0
